stdlib/3/subprocess.pyi

```diff
@@ -101,7 +101,7 @@ class Popen:
     def poll(self) -> int: ...
     def wait(self) -> int: ...
     # Return str/bytes
-    def communicate(self, input=...) -> Tuple[Any, Any]: ...
+    def communicate(self, input=..., timeout: int = ...) -> Tuple[Any, Any]: ...
```

Comment:
If this is new in Python 3.3 you should make it a conditional definition. Look around for examples usingof sys.version_info.